### PR TITLE
Check that error is a thing before formatting it.

### PR DESCRIPTION
**Pull request by WiseBird**
Sometimes `error` is `undefined` and you get `TypeError: Cannot read property 'module' of undefined`.
In my case it happened when I used `antlr4` package that in order used `fs` node packaged and actual error was `Module not found: Error: Can't resolve 'fs' in '...'`

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -189,7 +189,7 @@ WebpackBuildNotifierPlugin.prototype.onCompilationDone = function(results) {
         var error = results.compilation.errors[0];
         notify = true;
         title += 'Error';
-        msg = this.messageFormatter(error, error.module && error.module.rawRequest ? error.module.rawRequest : '');
+        msg = error && this.messageFormatter(error, error.module && error.module.rawRequest ? error.module.rawRequest : '');
```
Comment (RoccoC) [inline index.js]: I presume we should do the same thing for the next code block to handle null/undefined `warning` as well?

Also would be nice to include a default error/warning message in the case when `!error` or `!warning`.

**Comment by RoccoC** [comment]
In addition, can you please update your PR to bump the package version and add an entry to the changelog in README.md? Thanks!

File: index.js (hunk 1 above)
Comment (WiseBird) [reply index.js]: Done. Please check.

**Comment by RoccoC** [comment]
Merged, thank you! 👍

**Pull request merged**